tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -118,8 +150,54 @@ def fin():
     request.addfinalizer(fin)
 
 
+@pytest.fixture(scope='module')
+def create_project_client_ec2(request):
+    node_roles = [["controlplane"], ["etcd"],
+                  ["worker"], ["worker"], ["worker"]]
+    cluster, aws_nodes = create_and_validate_custom_host(node_roles)
+    client = get_user_client()
+    p, ns = create_project_and_ns(USER_TOKEN, cluster, "testsecret")
+    p_client = get_project_client_for_token(p, USER_TOKEN)
+    c_client = get_cluster_client_for_token(cluster, USER_TOKEN)
+    namespace["p_client"] = p_client
+    namespace["ns"] = ns
+    namespace["cluster"] = cluster
+    namespace["project"] = p
+    namespace["c_client"] = c_client
+
+    def fin():
+        client.delete(namespace["project"])
+        cluster_cleanup(client, cluster, aws_nodes)
+    request.addfinalizer(fin)
+
+
```

Comment:
create_project_client_and_cluster_local fixture is not getting used in any place?